app/decorators/event_decorator.rb

```diff
@@ -0,0 +1,24 @@
+class EventDecorator < Draper::Decorator
+  delegate_all
+
+   def self.collection_decorator_class
+    PaginatingDecorator
+  end
+
+  def beginning_at
+    object.beginning_at.strftime("%Y-%m-%d %R")
+  end
+
+  def end_at
+    object.beginning_at.strftime("%Y-%m-%d %R")
```

Comment:
as I wrote above if do so then you would end up with format written in one place (locales)